code/csharp-dotnet2/docs/Geslacht_enum.md

```diff
@@ -0,0 +1,8 @@
+# IO.Swagger.Model.GeslachtEnum
+## Properties
+
```

Comment:
Hoeven de waarden van een enumeratie niet gedfinieerd te worden  ?

Reply:
Dit is geen code file, maar een markdown file. En een enum kent geen properties. Daarom is de file leeg